feature-libs/product-configurator/common/components/configurator-cart-entry-info/configurator-cart-entry-info.component.spec.ts

```diff
@@ -239,5 +241,86 @@ describe('ConfiguratorCartEntryInfoComponent', () => {
         ).toBe(1);
       });
     });
+
+    describe('Accessibility', () => {
```

Comment:
Is it possible to rephrase that so that it makes sense read together with the test descriptions? 
Now it reads 'Accessibility should contain div element with class name....'

Reply:
Okay!